### PR TITLE
Define new code ownership based on SIGs

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This updates our OWNERS files to the new SIG structure.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -1,16 +1,38 @@
...
+    - zyanichaimaa
+
+  sig-cluster-management:
     - kron4eg
     - moadqassem
     - moelsayed
     - xmudrii
     - xrstf
+
+  sig-networking:
```
Comment (irozzo-1A) [inline OWNERS_ALIASES:24]: please add @rastislavs

Comment (xrstf) [reply OWNERS_ALIASES:24]: Oops, sorry, my bad. Fixed.

**Pull request merged**